src/commands/ft_info.cc

```diff
@@ -26,9 +26,9 @@
 namespace valkey_search {
 
 constexpr absl::string_view kFTInfoTimeoutMsConfig{"ft-info-timeout-ms"};
-constexpr uint32_t kDefaultFTInfoTimeoutMs{5000};
+constexpr uint32_t kDefaultFTInfoTimeoutMs{1000};
 constexpr uint32_t kMinimumFTInfoTimeoutMs{100};
-constexpr uint32_t kMaximumFTInfoTimeoutMs{300000};  // 5 minutes max
+constexpr uint32_t kMaximumFTInfoTimeoutMs{10000};  // 10s max
```

Comment:
Question: Why do we need a different timeout config for FT.INFO? Can we not use the existing timeout config?